src/index.html

```diff
@@ -15,8 +15,114 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo"
+        href="#"
+      >
+        <img
+          class="logo logo__img"
+          src="../src/images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              class="link_2"
+              href="#"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              class="link_2"
+              href="#"
+            >
+              SMATHPHONE
+            </a>
+          </li>
+          <li>
+            <a
+              class="link"
+              href="#"
+            >
+              LAPTOPS E COMPUTERS
+            </a>
+          </li>
+          <li>
+            <a
+              class="link_2"
+              href="#"
+            >
+              GARTGETS
```

Comment:
There is a typographical error in the word 'GARTGETS'. It should be corrected to 'GADGETS'.